src/h5web/visualizations/heatmap/HeatmapToolbar.module.css

```diff
@@ -0,0 +1,32 @@
+.toolbar {
+  display: flex;
+  padding: 0 0.5rem;
+  align-items: stretch;
+  font-size: 0.875rem;
+}
+
+.togglers {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.scaleTypeToggler,
+.aspectRatioToggler {
+  padding: 0 0.5rem;
+  composes: btn-clean from global;
+  display: flex;
+  align-items: center;
+  justify-content: left;
+}
+
+.scaleTypeToggler:hover,
+.scaleTypeToggler[aria-checked='true'],
+.aspectRatioToggler:hover,
+.aspectRatioToggler[aria-checked='true'] {
+  background-color: var(--primary-dark-bg);
+}
```

Comment:
This needs perhaps some CSS tidying-up :sweat_smile:

Reply:
I have to admit I'm not a huge fan of having the two togglers stacked up. Are you worried about horizontal space? Perhaps we could shorten the labels, like `Keep ratio`, and `Log scale` ?

Reply:
No, I am not worried. It was just a preliminary layout for discussion. I agree for `Log Scale` but I fear that `Keep ratio` is not as meaningful. Still, let's roll with that and see how it goes in the future.